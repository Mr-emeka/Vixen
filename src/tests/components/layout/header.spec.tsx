import React from "react";
import { shallow } from 'enzyme';
import Header from '../../../components/layout/header';

describe('Header Card', () => {
    it('should render correctly', () => {
        const HeaderContainer = shallow(<Header show={false} handleSearch={() => console.log('search')} />);
        const BrandContainer = HeaderContainer.find('.header__brand')
        expect(HeaderContainer).toMatchSnapshot();
    });
});