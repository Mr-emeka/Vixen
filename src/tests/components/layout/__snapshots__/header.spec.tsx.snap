// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Card should render correctly 1`] = `ShallowWrapper {}`;
